docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,354 @@
+``DT[, .N, by=x]``                                ``DT[:, dt.count(), by("x")]``                   Number of rows per group
+``DT[, .SD[1], by=x]``                            ``DT[0, :, by('x')]``                            | Select first row of ``y`` and ``v``
+                                                                                                   | for each group in ``x``
+``DT[, c(.N, lapply(.SD, sum)), by=x]``           ``DT[:, dt.sum(f[:]), by("x")]``                 | Get rows and sum columns ``v`` and ``y``
+                                                                                                   | by group
+``DT[, sum(v), by=.(y%%2)]``                      ``DT[:, dt.sum(f.v), by(f.y%2)]``                Expressions in :func:`by`
+
+``DT[, .SD[which.min(v)], by=x]``                 ``DT[0, f[:], by("x"), dt.sort(f.v)]``           Get row per group where column ``v`` is minimum
+
+``DT[, tail(.SD,2), by=x]``                       ``DT[-2:, :, by("x")]``                          Last 2 rows of each group
+========================================         =============================================    ============================================================
+
+In R's `data'table <https://data.table.gitlab.io/data.table/index.html>`__, the order of the groupings is preserved; in ``datatable``, the returned dataframe is sorted on the grouping column. ``DT[, sum(v), keyby=x]`` in data.table returns a dataframe ordered by column ``x``.
+
+Also, in ``datatable``, :ref:`f-expressions` in the ``i`` section of a groupby is not yet implemented, hence the chaining method to get the sum of column ``v`` where ``x!=a``.
+
+Multiple aggregations within a group can be executed in R's `data'table <https://data.table.gitlab.io/data.table/index.html>`__ with the syntax below ::
+
+    DT[, list(MySum=sum(v),
+              MyMin=min(v),
+              MyMax=max(v)),
+       by=.(x, y%%2)]
+
+The same can be replicated in ``datatable`` by using a dictionary ::
+
+    DT[:, {MySum: dt.sum(f.v),
+           MyMin: dt.min(f.v),
+           MyMax: dt.max(f.v)},
+       by(f.x, f.y%2)]
+
+
+Add/Update/Delete Columns
+-------------------------
+
+========================================         ===============================================  ============================================================
+data.table                                           datatable                                              Action
+========================================         ===============================================  ============================================================
+``DT[, z:=42L]``                                 | ``DT[:, update(z=42)]`` or                       Add new column
+                                                 | ``DT['z'] = 42`` or
+                                                 | ``DT[:, 'z'] = 42`` or
+                                                 | ``DT = DT[:, f[:].extend({"z":42})]``
```

Comment:
Also: create a new column whose name is given in a variable `var`: in python it's `DT[var] = 4242`.